.github/workflows/ci_cd.yml

```diff
@@ -162,6 +158,31 @@ jobs:
           name: api_logs
           path: api_logs
 
+  test-redoc:
+    name: Check for API consumer docs
+    runs-on: ubuntu-latest
+    needs:
+      - build-images
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Setup just
+        uses: extractions/setup-just@v1
+
+      - name: Download all images
+        uses: actions/download-artifact@v2
+        with:
+          path: /tmp
+
+      - name: Load all images
+        run: |
+          docker load --input /tmp/api/api.tar
+          docker load --input /tmp/ingestion_server/ingestion_server.tar
```

Comment:
I don't know if this is possible with GHA, but it would be really cool to have this as a set of steps that we could just reference as a precursor to specific tasks! Since this does appear to be reused elsewhere in this file.

Reply:
It is possible using YAML anchors but `actionlint` does not support them yet: https://github.com/rhysd/actionlint/issues/133